kafka-ui-react-app/src/redux/reducers/schemas/selectors.ts

```diff
@@ -24,6 +34,22 @@ export const getSchemaList = createSelector(
     if (!isFetched) {
       return [];
     }
-    return allNames.map((subject) => byName[subject]);
+    return allNames.map((subject) => byName[subject as string]);
   }
 );
+
+const getSchemaName = (_: RootState, subject: string) => subject;
```

Comment:
This selector seems redundant or incorrect, as it doesn't do anything.

Reply:
Actually not, We use it to get `subject`